_posts/2023-05-17-announcing-duckdb-080.md

```diff
@@ -0,0 +1,177 @@
+
+```sql
+CREATE TABLE bigdata(col INTEGER);
+INSERT INTO bigdata VALUES (NULL), (42), (NULL), (43);
+FROM bigdata ORDER BY col DESC LIMIT 3;
+```
+
+| v0.7.1 | v0.8.0 |
+|--------|--------|
+| NULL   | 43     |
+| NULL   | 42     |
+| 43     | NULL   |
+
+#### New SQL Features
+
+[**Pivot and Unpivot**](https://github.com/duckdb/duckdb/pull/6387). There are many shapes and sizes of data, and we do not always have control over the process in which data is generated. While SQL is well-suited for reshaping datasets, turning columns into rows or rows into columns is tedious in vanilla SQL. With this release, DuckDB introduces the `PIVOT` and `UNPIVOT` statements that allow reshaping data sets so that rows are turned into columns or vice versa. A key advantage of DuckDB's syntax is that the column names to pivot or unpivot can be automatically deduced. Here is a short example:
+
+```sql
+CREATE TABLE sales(year INT, amount INT);
+INSERT INTO sales VALUES (2021, 42), (2022, 100), (2021, 42);
+PIVOT sales ON year USING SUM(amount);
+```
+
+| 2021 | 2022 |
+|------|------|
+| 84   | 100  |
+
+The [documentation contains more examples](/docs/sql/statements/pivot.html).
+
+
+[**ASOF Joins for Time Series**](https://github.com/duckdb/duckdb/pull/6719). When joining time series data with background fact tables, the timestamps often do not exactly match. In this case it is often desirable to join rows so that the timestamp is joined with the *nearest timestamp*. The ASOF join can be used for this purpose - it performs a fuzzy join to find the closest join partner for each row instead of requiring an exact match.
+
+
+```sql
+CREATE TABLE a(ts TIMESTAMP);
+CREATE TABLE b(ts TIMESTAMP);
+INSERT INTO a VALUES (TIMESTAMP '2023-05-15 10:31:00'), (TIMESTAMP '2023-05-15 11:31:00');
+INSERT INTO b VALUES (TIMESTAMP '2023-05-15 10:30:00'), (TIMESTAMP '2023-05-15 11:30:00');
+
+FROM a ASOF JOIN b ON a.ts >= b.ts;
+```
+
+|        a.ts         |        b.ts         |
+|---------------------|---------------------|
+| 2023-05-15 10:31:00 | 2023-05-15 10:30:00 |
+| 2023-05-15 11:31:00 | 2023-05-15 11:30:00 |
+
+
+Please [refer to the documentation](/docs/guides/sql_features/asof_join.html) for a more in-depth explanation.
+
+
+
+#### Data Integration Improvements
+
+[**Default Parallel CSV Reader.**](https://github.com/duckdb/duckdb/pull/6977). In this release, the parallel CSV reader has been vastly improved and is now the default CSV reader. We would like to thank everyone that has tried out the experimental reader for their valuable feedback and reports. The `experimental_parallel_csv` flag has been deprecated and is no longer required. The parallel CSV reader enables much more efficient reading of large CSV files. 
```

Comment:
```suggestion
[**Default Parallel CSV Reader**](https://github.com/duckdb/duckdb/pull/6977). In this release, the parallel CSV reader has been vastly improved and is now the default CSV reader. We would like to thank everyone that has tried out the experimental reader for their valuable feedback and reports. The `experimental_parallel_csv` flag has been deprecated and is no longer required. The parallel CSV reader enables much more efficient reading of large CSV files. 
```

Reply:
Spatial had its own blog post, which is why I left it out here